api/app/session_service.py

```
@@ -0,0 +1,405 @@
+    JWT secret key is fetched from AWS Secrets Manager and cached
+    for performance. Supports automatic key rotation.
+    """
+
+    # Class-level cache for JWT secret (shared across instances)
+    _jwt_secret_cache: Optional[Dict[str, Any]] = None
+    _jwt_secret_cache_time: Optional[float] = None
+    _jwt_secret_cache_ttl: int = 3600  # Cache for 1 hour
+
+    def __init__(
+        self,
+        sessions_table_name: str = None,
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)